ezbake/config/conf.d/auth.conf

```diff
@@ -56,18 +56,43 @@ authorization: {
             name: "puppetlabs environments"
         },
         {
-            # Allow nodes to access all file services; this is necessary for
-            # pluginsync, file serving from modules, and file serving from
-            # custom mount points (see fileserver.conf). Note that the `/file`
-            # prefix matches requests to file_metadata, file_content, and
-            # file_bucket_file paths.
+            # Allow nodes to access all file_bucket_files.  Note that access for
+            # the 'delete' method is forbidden by Puppet regardless of the
+            # configuration of this rule.
             match-request: {
-                path: "/puppet/v3/file"
+                path: "/puppet/v3/file_bucket_file"
                 type: path
+                method: [get, head, post, put]
             }
             allow: "*"
             sort-order: 500
-            name: "puppetlabs file"
+            name: "puppetlabs file bucket file"
+        },
+        {
+            # Allow nodes to access all file_content.  Note that access for the
+            # 'delete' method is forbidden by Puppet regardless of the
+            # configuration of this rule.
+            match-request: {
+                path: "/puppet/v3/file_content"
+                type: path
+                method: [get, post]
+            }
+            allow: "*"
+            sort-order: 500
+            name: "puppetlabs file content"
+        },
+        {
+            # Allow nodes to access all file_metadata.  Note that access for the
+            # 'delete' method is forbidden by Puppet regardless of the
+            # configuration of this rule.
+            match-request: {
+                path: "/puppet/v3/file_metadata"
```

Comment:
It will, yeah.  With the `type: path` qualifier, the value for `path` is basically treated as a "starts-with" for matching purposes.